### PR TITLE
Kong 2.1.0 beta.1

### PR DESCRIPTION
**Pull request by hutchic**
Kong 2.1.0-beta.1 is available.
Also pruned some older versions

### CODE REVIEW

**Comment by yosifkit** [comment]
https://github.com/Kong/docker-kong/blob/69c50b396440ab6434853b982c3295d54a9fb3f2/ubuntu/Dockerfile#L18-L30

I am slightly confused as to why the `|| true` is needed for either of these packages on `ubuntu:xenial` (since they [both](https://packages.ubuntu.com/xenial/zlibc) [exist](https://packages.ubuntu.com/xenial/zlib1g-dev)).

In the unlikely event of failure, there *could* be a bug here since and (`&&`) binds tighter than or (`||`) so this block becomes roughly this:

```dockerfile
RUN ... \
    ( apt-get update \
    && apt-get install -y --no-install-recommends perl unzip git \
    && apt-get install -y --no-install-recommends zlibc ) || ( true \
    && apt-get install -y --no-install-recommends zlib1g-dev ) || ( true \
    && rm -rf /var/lib/apt/lists/* \
	&& dpkg -i /tmp/kong.deb \
	&& rm -rf /tmp/kong.deb \
	&& useradd -ms /bin/bash kong \
    && mkdir -p "/usr/local/kong" \
	&& chown -R kong:0 /usr/local/kong \
	&& chown kong:0 /usr/local/bin/kong \
	&& chmod -R g=u /usr/local/kong \
	&& kong version )
```

One solution is to switch all `&&` to semicolons like the if statement; the other solution is to use curly braces for grouping:  `&& { apt-get install -y --no-install-recommends zlibc || true; } \`.

**Comment by hutchic** [comment]
we recycle the same Dockerfile for bionic and focal. I'll try to adjust the line in question

File: library/kong [hunk 1]
```diff
@@ -38,62 +56,6 @@ GitFetch: refs/tags/1.5.1
...
-GitCommit: 196331b1e6b4798032af4d6c218a441e2c8db74d
-GitFetch: refs/tags/1.1.3
-Directory: alpine
-
-Tags: 1.1.3-centos, 1.1-centos
-GitCommit: 196331b1e6b4798032af4d6c218a441e2c8db74d
-GitFetch: refs/tags/1.1.3
-Directory: centos
-
 Tags: 1.0.4-alpine, 1.0.4, 1.0
```
Comment (gszr) [inline library/kong:59]: Let's also remove these 1.0.4 from the list.

Comment (yosifkit) [reply library/kong:59]: Should we delay this PR or can this removal be addressed in another PR?

**Pull request merged**